irsa.tf

```
@@ -0,0 +1,41 @@
+locals {
+  create_namespace = var.irsa_config.create_kubernetes_namespace && var.irsa_config.kubernetes_namespace != "kube-system"
+}
+
+module "irsa_role" {
+  source  = "terraform-aws-modules/iam/aws//modules/iam-role-for-service-accounts-eks"
+  version = ">= 5.5.0"
```

```suggestion
  version = "~> 5.5.0"
```

Best not to do this. Prone to breakages.